drivers/hmis/lib/form_data/tarrant_county/ce_referral_steps/ce_shelter_referral.json

```diff
@@ -427,13 +405,6 @@
                   "custom_field_key": "ce_shelter_referral_individual_id_dl_or_passport"
                 },
                 "component": "CHECKBOX",
-                "enable_when": [
-                  {
-                    "operator": "EQUAL",
-                    "question": "family_or_individual",
-                    "answer_code": "Individual"
```

Comment:
No change in behavior, these were redundant because the parent group item already has the same rule